docs/dev.md

```diff
@@ -0,0 +1,117 @@
+Development
+===========
+This page contains information for developers about how to contribute to this project.
+
+Set-up
+------
+This project can be used in combination with  [dans-dev-tools]{:target=_blank}. 
+Before you can start it as a service some dependencies must first be started.
+One depency is [dd-validate-dans-bag].
+For the other dependency you have the choice between a local service [dd-vault-catalog] and a Virtual Machine (VM) `dev_transfer`.
+The local variant requires a local database. 
+For the VM you need access to the project [dd-dtap]{:target=_blank},
+
+[dans-dev-tools]: https://github.com/DANS-KNAW/dans-dev-tools#readme
+[dd-dtap]: https://github.com/DANS-KNAW/dd-dtap#readme
+[dd-validate-dans-bag]: https://github.com/DANS-KNAW/dd-validate-dans-bag#readme
+[dd-vault-catalog]: https://github.com/DANS-KNAW/dd-vault-catalog#readme
+
+### Initialize development environment
+
+This is only necessary once per project. If you execute this any existing configuration and data will be reset.
+
+Open separate terminal tabs for `dd-vault-ingest-flow`, its dependency `dd-validate-dans-bag`
+and the optional dependency `[dd-vault-catalog`. In each tab run:
+
+```commandline
+start-env.sh
+```
+
+The service `dd-validate-dans-bag` needs different configurations for `dd-vault-ingest-flow` and other services. 
+So you will have to update the generated `dd-validate-dans-bag/etc/config.yml`,
+perhaps keep copies of the files or comment lines to switch between both situations.
+
+* remove one level from `../../` in the schema locations.
```

Comment:
I only see ../dans-schema, so it might not be needed

Reply:
That change was pretty recent.